docs/api/version-history.md

```diff
@@ -20,6 +20,7 @@ keywords: "API, Docker, rcli, REST, documentation"
 * `GET /containers/(id or name)/attach/ws` now returns WebSocket in binary frame format for API version >= v1.27, and returns WebSocket in text frame format for API version< v1.27, for the purpose of backward-compatibility.
 * `GET /networks` is optimised only to return list of all networks and network specific information. List of all containers attached to a specific network is removed from this API and is only available using the network specific `GET /networks/{network-id}.
 * `GET /containers/json` now supports `publish` and `expose` filters to filter containers that expose or publish certain ports.
+* `GET /secrets` now does not support filter `names`.
```

Comment:
As it is supported in `1.13` release (the `names` filter), we can't remove it *just like that*. We need to do what our deprecation policy requires us to do (i.e. mark it as deprecated for next release, and will be remove in 3 more releases)
/cc @thaJeztah